foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -58,6 +59,8 @@
 
   private static final String MICROSERVICE_CONFIG_LOADER_KEY = "cse-microservice-config-loader";
 
+  private static final String IS_PRINT_URL = "cse.config.isPrintUrl";
```

Comment:
Use "servicecomb.config.log.verbose" is better, and can control other logs in future.

Reply:
done